Source/Scene/TileMapServiceImageryProvider.js

```diff
@@ -385,7 +385,7 @@ TileMapServiceImageryProvider.prototype._metadataFailure = function (error) {
   const minimumLevel = calculateSafeMinimumDetailLevel(
     tilingScheme,
     rectangle,
-    options.maximumLevel
+    options.minimumLevel
```

Comment:
So this was changed from `options.minimumLevel` to `options.maximumLevel` in what looks to be a regression when we formatted the codebase with prettier in 2fd0e8f7e4212bd1e7084299187f70597a6bbfd8. So this change looks good. 

One thing I'm confused about though, is that the unit tests did not change as part of the above regression, and were originally using `maximumLevel` in both places. But your changes make sense to me, and I think they may have been passing by lucking into the default values.

Reply:
I see. From context though, I agree that these changes all look straightforward.